tests/cases/care_partner_platform/case_care_platform_participants.yaml

```diff
@@ -3985,261 +3985,44 @@ TestCarePartnerParticipantReviewEditOrAddVendorInfo:
+  Actions:
+  # Calling Login case.
     - Type: case
-      Value: CarePartnerToastValidation
-  # Validate if the new change is applied
-    - Type: get_text
-      Strategy: xpath
-      Id : $PAGE.care_platform_edit_participants_vendor.name_vendor_record$
-      Greps:
-        - var: CATEGORY_VENDOR_NAME
-          match: "(.*)"
-    - Type: assert
-      Role: desktopChrome
-      Asserts:
-        - Type: eq
-          Var: CATEGORY_VENDOR_NAME
-          Value: $AND_CLI_NEW_NAME$
-  # Delete category 
+      Value: CarePartnerLogin
+  # looking for a senior profile and click on it
     - Type: click
-      Strategy: xpath
-      Id: $PAGE.care_platform_edit_participants_vendor.delete_created_vendor_button$
-    - Type: wait_for
-      Strategy: id
-      Id: $PAGE.care_platform_edit_participants_vendor.vendor_dialog$
-  # Validate if "Delete Vendor" is opened
-      Vars:
-        MODAL_NAME: Delete Vendor
-    - Type: get_text
-      Strategy: xpath
-      Id: $PAGE.care_platform_edit_participants_vendor.dialog_title_label$
-      Greps:
-        - var: MODAL_TITLE
-          match: "(.*)"
-    - Type: assert
-      Role: desktopChrome
-      Asserts:
-        - Type: eq
-          Var: MODAL_TITLE
-          Value: $AND_CLI_MODAL_NAME$
-  # Closing "Delete Vendor" dialog
+      Strategy: xpath 
+      Id: $PAGE.care_platform_participant.search_button$
+    - Type: send_keys
+      Strategy: xpath 
+      Id: $PAGE.care_platform_participant.search_input$
+      Value: $AND_CLI_senior_user1_name$
     - Type: click
       Strategy: xpath
-      Id: $PAGE.care_platform_edit_participants_vendor.cancel_vendor_dialogs_button$
-  # Checking if vendor is not deleted
-    - Type: wait_for
+      Id: $PAGE.care_platform.navigation_participant$
+    - Type: click 
       Strategy: xpath
-      Id: $PAGE.care_platform_edit_participants_vendor.name_vendor_record$
-  # Open "Delete Vendor" dialog
+      Id: $PAGE.care_platform_participant.first_participant_row$
+  # Click on "Edit Info" button 
     - Type: click
+      Strategy: xpath 
+      Id: //button[text()='Edit Info']
+  # go to "Vendor" seccion and click on it.
+    - Type: scroll_to
       Strategy: xpath
-      Id: $PAGE.care_platform_edit_participants_vendor.delete_created_vendor_button$
-    - Type: wait_for
-      Strategy: id
-      Id: $PAGE.care_platform_edit_participants_vendor.vendor_dialog$
-  # Confirm delete vendor
-    - Type: wait_for
-      Strategy: id
-      Id: dialogs
+      Id: //ul[@class='form-navigation']//li/button[text()='Vendors']
     - Type: click
       Strategy: xpath
-      Id: $PAGE.care_platform_edit_participants_vendor.delete_confirmation_modal$
-  # Wait to close dialog
-    - Type: wait_not_visible
-      Strategy: xpath
-      Id: $PAGE.care_platform_edit_participants_vendor.vendor_dialog$
-  # Validate toast message
-      Vars:
-        TOAST_MESSAGE_SHOWN: Vendor deleted
+      Id: //ul[@class='form-navigation']//li/button[text()='Vendors']
```

Comment:
this too

Reply:
got fix in this commit https://github.com/neveralone-chs-org/TestRay/pull/161/commits/4e54cbdd216d6d18a83351b7699e77d869968dc5